content/blog/2018-12-28-subquery.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Subquery
+created_at: 2018-12-28
+kind: article
+author_name: Ganesh Vernekar
+---
+
+## Introduction
+
+As the title suggests, subquery is a part of a query, and allows you to do a range query within a query, which was not possible before. It has been a long standing request on Prometheus [prometheus/prometheus/1227](https://github.com/prometheus/prometheus/issues/1227) and a desired feature.
```

Comment:
subquery -> a subquery

I would just say "long-standing feature request" and omit the "and a desired feature" and also omit "on Prometheus" (I think "on" isn't quite correct).